thorlcr/graph/thgraphmaster.cpp

```diff
@@ -713,12 +715,10 @@ cost_type CMasterActivity::calcFileReadCostStats(bool updateFileProps)
     {
         IDistributedFile *file = queryReadFile(0);
         if (file)
-        {
-            bool useJhtreeCache = (TAKindexread == container.getKind());
-            totalReadCost = calcReadCost(useJhtreeCache, *file, statsCollection);
-        }
+            // note: use jhtree cache stats to calculate file access cost if it is an index activity
```

Comment:
trivial/style: I think clearer to keep block braces when multiline block, even if not strictly needed, because only 1 of the lines is a non-comment ..

Reply:
agreed